code/game/objects/items/weapons/manuals.dm

```diff
@@ -168,6 +168,219 @@
 
 		"}
 
+/obj/item/book/manualshield_generator_guide
+	name = "Artificers Guild Shield Operating Guide"
+	icon_state = "book_shield"
+	author = "Jeanne Petite"
```

Comment:
Do you have permission to use this character?

Reply:
The book was given to me by Hound/Jeanne Petite about 2 years ago and at the time I said I'd add it in as a book. It slipped my mind until now, but consent for the such has not been retracted.

Reply:
👍 
Alrighto